psh/test-mkdir.py

```diff
@@ -0,0 +1,143 @@
+import random
+import string
+
+from collections import namedtuple
+
+File = namedtuple('File', ['name', 'owner', 'is_dir'])
+
+def ls(p, dir=''):
+    files = []
+
+    p.sendline(f'ls -la {dir}')
+    p.expect_exact('ls -la')
+    if dir:
+        p.expect_exact(f' {dir}')
+
+    p.expect_exact('\n')
+
+    while True:
+        idx = p.expect(['\(psh\)\% ', '(.*)(\r+)\n'])
+        if idx == 0:
+            break
+
+        line = p.match.group(0)
+
+        permissions, _, owner, _, _, _, _, _, name = line.split()
```

Comment:
Maybe this line in the block `try ... except`

Reply:
Exception will be raised. It will be changed in the next PR with assertion handling.